spec/models/piece_spec.rb

```diff
@@ -94,4 +94,37 @@ def create_chess(x, y)
       end
     end
   end
+
+  describe '#on_board?' do
+    it 'should return true is the coords are from 1 to 8' do
+      piece = Piece.create
+      (1..8).each do |x|
```

Comment:
Good test.

Readability nitpick. I'd throw a new line after this line `piece = Piece.create`.

Makes the test read more like "paragraphs". Separating the data creation and the expectations.

Makes code easier to skim when using white space to separate.